### ISSUE TITLE
panic when piped to `head`

### ISSUE THREAD

**Issue by Chaz6** [nachoparker/dutree#26]
```
$ dutree --version
dutree version v0.2.9
$ dutree -s | head -n 1
Unable to get terminal size
[ git 19.28 GiB ]
thread 'main' panicked at 'failed printing to stdout: Broken pipe (os error 32)', src/libstd/io/stdio.rs:792:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
```

**Comment by Akito13** [comment]
@Chaz6 

This happens because the pipe is closed by `head`, while dutree still tries writing to this closed pipe. That's what SIGPIPE is for, so you can catch the non-existent pipe and handle the exception.

**Comment by nachoparker** [comment]
fixed in `v0.2.12`
